rest/post_processing/gtiff_parser.py

```diff
@@ -10,37 +10,46 @@
 from processing.const import CustomMimeType
 from openeoerrors import Internal
 
+
 # assume it's only 1 time and 1 bands dimension
 def check_dimensions(time_dimensions, bands_dimensions):
-    if len(time_dimensions) == 0:
-        raise Internal("No time dimensions exist. Only 1 time dimension is supported.")
-
     if len(time_dimensions) > 1:
-        raise Internal("More than 1 time dimension exist. Only 1 time dimension is supported.")
-
-    if len(bands_dimensions) == 0:
-        raise Internal("No bands dimensions exist. Only 1 bands dimension is supported.")
+        raise Internal("More than 1 time dimension exist. Only 0 or 1 time dimension is supported.")
 
     if len(bands_dimensions) > 1:
-        raise Internal("More than 1 bands dimension exist. Only 1 bands dimension is supported.")
+        raise Internal("More than 1 bands dimension exist. Only 0 or 1 bands dimension is supported.")
```

Comment:
I find these 2 sentences weird and I am not sure if it is like that just because I read them 5000 times in the last 20min or what 😂 Don't know exactly how to change them but I would definitely add an `s` in `exist` at the end so it becomes `exists`.

Reply:
These error messages weren't well-worded, I agree :D

Should be better now

Reply:
Looks good. 🥇